lib/resolveDeps.js

```diff
@@ -32,7 +32,7 @@ function resolveDeps(ast, result) {
   } = result.opts;
 
   const cwd = dirname(selfPath);
-  const rootDir = dirname(rootPath);
+  const rootDir = rootPath;
```

Comment:
Not sure that I like that change, cause it can be misleading due to the variable name. 

I guess it's better to pass the `root` option with other opts to resolveDeps

```js
resolveDeps(ast, {opts: {from: rootPath, graph, resolve, root, rootPath, rootTree}, processor});
```

and make resolving based on opts:

```js
  const importNotes = comment({
    parent: rootTree,
    raws: {before: rootTree.nodes.length === 0 ? '' : '\n\n'},
    text: ` imported from ${normalizeUrl(relative(root || rootDir, selfPath))} `,
  });
```